bugwarrior/services/gitlab.py

```diff
@@ -433,7 +431,21 @@ def include_todo(todo):
 
     def issues(self):
         tmpl = '{scheme}://{host}/api/v4/projects'
-        all_repos = self._fetch_paged(tmpl)
+
+        all_repos = []
+        if self.include_repos:
```

Comment:
I think we should make this `if self.include_repos and not self.include_regex` to avoid doing these extra API calls if we're just going to throw out the results and fetch all the repos anyway.

Reply:
Updated in 849494e